### ISSUE TITLE
Unable to create character with a space in the name

### ISSUE THREAD

**Issue by Ruin0x11** [ElonaFoobar/ElonaFoobar#271]
# Environment

* OS version: Windows 7
* Elona Foobar verision: `develop`

# Details
You cannot create a character that has a space in the name, as it fails with this error:

```
boost::filesystem::copy_file: The system cannot find the path specified: "e:/build\ElonaFoobar\bin\debug\tmp\mdata_7_101.s2", "e:/build\ElonaFoobar\bin\debug\save\sav_ase asd \mdata_7_101.s2"
```

# How to reproduce
1. Attempt to create a character with a space in the name.

**Comment by ki-foobar** [comment]
On my environment, this issue occurs.

OS version: Windows 10 Home 1709
Elona Foobar version: 328a5646fdc5aa2841ca9c132026e935b1b1fd5b

I tried these names:
- "abc"(a, b, c): ok
- "a b c d e"(a, space, b, space, c, space, d, space, e): ok
- "ase asd "(a, s, e, space, a, s, d, space): error
- "a "(a, space): error

I tried also them in Japanese mode, then the results are the same.

Probably the last space cause the issue.
